Alignment/TrackerAlignment/plugins/TkAlCaOverlapTagger.cc

```diff
@@ -133,12 +142,12 @@ void TkAlCaOverlapTagger::produce(edm::Event& iEvent, const edm::EventSetup& iSe
             //
             AlignmentClusterFlag hitflag(hit->geographicalId());
             hitflag.SetOverlapFlag();
-            // cout<<"Overlap found in SubDet "<<subDet<<"!!!"<<flush;
+            LogDebug("TkAlCaOverlapTagger") << "Overlap found in SubDet " << subDet << "!!!" << std::flush;
```

Comment:
I think the `std::flush` is not needed, the edm::LogPrint will take care of it, no?